### PR TITLE
{tools}[gcccuda/2020b] gpustat v0.6.0 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3300.joltik.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz (cascadelake), Python 3.6.8
See https://gist.github.com/fc166cdfa5f3cc1f160b7a074a837225 for a full test report.

File: easybuild/easyconfigs/g/gpustat/gpustat-0.6.0-gcccuda-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+dependencies = [
+    ('Python', '3.8.6'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('psutil', '5.8.0', {
+        'checksums': ['0c9ccb99ab76025f2f0bbecf341d4656e9c1351db8cc8a03ccd62e318ab4b5c6'],
+    }),
```
Comment (branfosj) [inline easybuild/easyconfigs/g/gpustat/gpustat-0.6.0-gcccuda-2020b.eb:20]: We should be able to remove this. We have psutil 5.7.2 in the Python install and this version of gpustat was released long before psutil 5.8.0 was released.

**Review by branfosj — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3301.joltik.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz (cascadelake), Python 3.6.8
See https://gist.github.com/c6a21ae84bd64583dcbdff516339950d for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bber0501u03a.bb2.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/0be7ee4d8a68c4871572c11ccddf058d for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**